packages/accordions/examples/advanced.md

```diff
@@ -0,0 +1,205 @@
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { Toggle, Field, Label } = require('@zendeskgarden/react-forms/src');
+const { IconButton } = require('@zendeskgarden/react-buttons/src');
+const GearIcon = require('@zendeskgarden/svg-icons/src/16/gear-stroke.svg').default;
+const FolderIcon = require('@zendeskgarden/svg-icons/src/16/folder-open-stroke.svg').default;
+const { Dropdown, Menu, Item, Trigger } = require('@zendeskgarden/react-dropdowns/src');
+
+const StyledHint = styled.span`
+  font-size: ${props => props.theme.fontSizes.sm};
+  font-weight: normal;
+  font-style: italic;
+`;
+
+const StyledButtonGroup = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  width: ${props => props.theme.space.base * 28}px;
+`;
+
+const AdvancedExample = () => {
+  const [isCollapsible, setIsCollapsible] = React.useState(false);
+  const [isExpandable, setIsExpandable] = React.useState(false);
+  const [isBare, setIsBare] = React.useState(false);
+  const [isCompact, setIsCompact] = React.useState(false);
+
+  const buttonGroup = (
+    <StyledButtonGroup>
+      <Dropdown onSelect={item => alert(item)}>
+        <Trigger>
+          <IconButton aria-label="Settings" title="Settings">
+            <GearIcon />
+          </IconButton>
+        </Trigger>
+        <Menu placement="top" hasArrow>
+          <Item value="option-1">Option 1</Item>
+          <Item value="option-2">Option 2</Item>
+          <Item value="option-3">Option 3</Item>
+        </Menu>
+      </Dropdown>
+      <Dropdown onSelect={item => alert(item)}>
+        <Trigger>
+          <IconButton aria-label="Settings" title="Settings">
+            <FolderIcon />
+          </IconButton>
+        </Trigger>
+        <Menu placement="top" hasArrow>
+          <Item value="option-1">Option 1</Item>
+          <Item value="option-2">Option 2</Item>
+          <Item value="option-3">Option 3</Item>
+        </Menu>
+      </Dropdown>
+    </StyledButtonGroup>
+  );
+
+  return (
+    <Grid>
+      <Well isRecessed>
+        <Row>
+          <Col>
+            <Field>
+              <Toggle checked={isCollapsible} onChange={event => setIsCollapsible(!isCollapsible)}>
+                <Label>Collapsible</Label>
+              </Toggle>
+            </Field>
+            <Field className="u-mt">
+              <Toggle checked={isExpandable} onChange={event => setIsExpandable(!isExpandable)}>
+                <Label>Expandable</Label>
+              </Toggle>
+            </Field>
+            <Field className="u-mt">
+              <Toggle checked={isBare} onChange={event => setIsBare(!isBare)}>
+                <Label>Bare</Label>
+              </Toggle>
+            </Field>
+          </Col>
+        </Row>
+      </Well>
+      <Row className="u-mt">
+        <Col>
+          <Accordion
+            level={3}
+            isBare={isBare}
```

Comment:
@jzempel @hzhu for the sake of visual proof, i am not opposed to adding a toggle option for `isCompact`. 

as a design guideline, we don't want to encourage `isCompact` usage when there are many elements within the header i.e. icons and media items. we initially decided not to include it as it might unintentionally give the wrong idea about this guideline.

Reply:
The `isCompact` toggle has been added to the advanced example.